app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -292,6 +293,11 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
                 }
             }
         }
+
+        val startTime = SystemClock.elapsedRealtimeNanos()
+        components.core.engine.profiler?.addMarker("TestMarkerTime")
```

Comment:
While this is probably a non-issue, I'm wondering, if we used the function that uses a time value if it would change the time measurement. I'm looking at this: https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/gecko/GeckoJavaSampler.java#144

When there's a `startTime` provided but no `endTime` we'd end up calling into the profiler code and i wonder how much overhead that would add? or maybe that doesn't matter if we're not thinking of adding time readings to our markers.

Reply:
I'm not too concerned about the specific time in this instance but that's good to keep in mind for future calls (e.g. ones being used for CI start up measurements).